CHANGELOG.next.toml

```
@@ -22,6 +22,32 @@ references = ["aws-sdk-rust#560", "smithy-rs#1487"]
 meta = { "breaking" = false, "tada" = false, "bug" = true }
 author = "rcoh"
 
+[[smithy-rs]]
+message = """
+Replaced use of `pin-project` with equivalent `pin-project-lite`. For pinned enum tuple variants and tuple structs, this
+change requires that we switch to using enum struct variants and regular structs. Most of the structs and enums that
+were updated had only private fields/variants and so have the same public API. However, this change does affect the
+public API of `aws_smithy_http_tower::map_request::MapRequestFuture<F, E>`. The `Inner` and `Ready` variants contained a
+single value. Each have been converted to struct variants and the inner value is now accessible by the `inner` field
+instead of the `0` field.
+"""
+references = ["smithy-rs#932"]
+meta = { "breaking" = true, "tada" = false, "bug" = false }
+author = "Velfi"
+
+[[aws-sdk-rust]]
+message = """
+Replaced use of `pin-project` with equivalent `pin-project-lite`. For pinned enum tuple variants and tuple structs, this
+change requires that we switch to using enum struct variants and regular structs. Most of the structs and enums that
+were updated had only private fields/variants and so have the same public API. However, this change does affect the
+public API of `aws_smithy_http_tower::map_request::MapRequestFuture<F, E>`. The `Inner` and `Ready` variants contained a
+single value. Each have been converted to struct variants and the inner value is now accessible by the `inner` field
+instead of the `0` field.
+"""
+references = ["smithy-rs#932"]
+meta = { "breaking" = true, "tada" = false, "bug" = true }
```

This wasn't matching the other entry. Not sure which you intended.
```suggestion
meta = { "breaking" = true, "tada" = false, "bug" = false }
```